src/components/InventoryDetail/ApplicationDetails.js

```diff
@@ -29,6 +32,16 @@ const ApplicationDetails = ({ onTabSelect, appList, ...props }) => {
         }
     }, []);
 
+    useEffect(() => {
+        const filteredResult = disabledApps && disabledApps.length && applications.filter(app => app.name !== disabledApps[0]);
```

Comment:
This can be simplified with safe accessor, using `includes` and fallbacking to original value
```suggestion
        // filters out disabled apps, if no disabledApps is used it will use the full applications list
        const filteredResult = applications.filter(app => !disabledApps?.includes(app.name));
        setActiveTabs(filteredResult)
```